src/openrct2/rct2/S6Importer.cpp

```diff
@@ -670,7 +670,7 @@ IParkImporter * ParkImporter::CreateS6(IObjectRepository * objectRepository, IOb
 
 extern "C"
 {
-    ParkLoadResult * game_load_sv6_path(const char * path)
+    ParkLoadResult * load_from_sv6(const char * path)
```

Comment:
On second thought - if I make the `load_from_sxx()` functions part of the ParkImporter interface then I'll be adding two separate functions (load save & load scenario) to said interface.

Given that our custom save format won't have this distinction I wasn't sure whether it was a good idea after all. Of course, we could just have the interface implementation for our format work in such a way that one function just calls the other but...